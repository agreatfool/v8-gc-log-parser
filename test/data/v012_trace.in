[88872]    90523 ms: Scavenge 1132.1 (1214.5) -> 1119.2 (1215.5) MB, 1.5 ms [allocation failure].
[88872]    90612 ms: Mark-sweep 1130.7 (1215.5) -> 1115.4 (1217.5) MB, 10.3 ms (+ 14.3 ms in 21 steps since start of marking, biggest step 3.5 ms) [GC interrupt] [GC in old space requested].
[88872]   111427 ms: Mark-sweep 1357.4 (1444.5) -> 1357.4 (1444.5) MB, 38.7 ms [last resort gc].
