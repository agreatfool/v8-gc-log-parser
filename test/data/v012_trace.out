[
  {
    "pid": 88872,
    "version": "v0.12.x",
    "time_since_init": 90523,
    "data": {
      "collector": "scavenge",
      "start_object_size": 1132.1,
      "end_object_size": 1119.2,
      "start_memory_size": 1214.5,
      "end_memory_size": 1215.5,
      "pause": 1.5,
      "gc_reason": "allocation failure"
    },
    "type": "trace"
  },
  {
    "pid": 88872,
    "version": "v0.12.x",
    "time_since_init": 90612,
    "data": {
      "collector": "mark_sweep",
      "start_object_size": 1130.7,
      "end_object_size": 1115.4,
      "start_memory_size": 1215.5,
      "end_memory_size": 1217.5,
      "pause": 10.3,
      "step_description": "+ 14.3 ms in 21 steps since start of marking, biggest step 3.5 ms",
      "gc_reason": "GC interrupt",
      "collector_reason": "GC in old space requested"
    },
    "type": "trace"
  },
  {
    "pid": 88872,
    "version": "v0.12.x",
    "time_since_init": 111427,
    "data": {
      "collector": "mark_sweep",
      "start_object_size": 1357.4,
      "end_object_size": 1357.4,
      "start_memory_size": 1444.5,
      "end_memory_size": 1444.5,
      "pause": 38.7,
      "gc_reason": "last resort gc"
    },
    "type": "trace"
  }
]